core/src/main/scala/kafka/server/DynamicConfig.scala

```diff
@@ -32,19 +33,25 @@ object DynamicConfig {
 
   object Broker {
     //Properties
-    val ThrottledReplicationRateLimitProp = "replication.quota.throttled.rate"
+    val ThrottledLeaderReplicationRateProp = "leader.replication.throttled.rate"
+    val ThrottledFollowerReplicationRateProp = "follower.replication.throttled.rate"
 
     //Defaults
-    val DefaultThrottledReplicationRateLimit = ReplicationQuotaManagerConfig.QuotaBytesPerSecondDefault
+    val DefaultThrottledReplicationRate = ReplicationQuotaManagerConfig.QuotaBytesPerSecondDefault
 
     //Documentation
-    val ThrottledReplicationRateLimitDoc = "A long representing the upper bound (bytes/sec) on replication traffic for replicas enumerated in the " +
-      s"property $ThrottledReplicationRateLimitProp. This property can be only set dynamically via the config command etc. The minimum value is 1 KB/s."
+    val ThrottledLeaderReplicationRateDoc = "A long representing the upper bound (bytes/sec) on replication traffic for leaders enumerated in the " +
+      s"property ${LogConfig.LeaderThrottledReplicasListProp} (for each topic). This property can be only set dynamically (via the kafka-configs.sh etc). It is suggested that the " +
+      s"limit be kept above the value of ${KafkaConfig.ReplicaFetchMaxBytesProp} which defaults to ${Defaults.ReplicaFetchMaxBytes}B/s"
```

Comment:
replaced with "limit be kept above 1MB/s for accurate behaviour." 
let me know if you'd like to tweak further.